app/src/main/java/com/fs_sournary/weather/screen/main/fragment/forecast/ForecastModule.java

```diff
@@ -0,0 +1,59 @@
+package com.fs_sournary.weather.screen.main.fragment.forecast;
+
+import android.support.v4.app.Fragment;
+
+import com.fs_sournary.weather.data.WeatherRepository;
+import com.fs_sournary.weather.data.model.DailyWeather;
+import com.fs_sournary.weather.data.source.remote.WeatherRemoteDataSource;
+import com.fs_sournary.weather.data.source.remote.api.service.WeatherApi;
+import com.fs_sournary.weather.utils.ConvertTimeUtils;
```

Comment:
ERROR: Unused import: com.fs_sournary.weather.utils.ConvertTimeUtils. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports